api/src/bs/Launch/updateAvgDiffPercent.js

```diff
@@ -0,0 +1,16 @@
+const { connectDb } = require("../../knex/knex");
+
+// should be called after screenshot is finished processing
+module.exports = async ({ launchId }) => {
+  const knex = await connectDb();
+  const avg = await knex("Screenshot")
+    .avg("diffPercentage")
+    .where({ launchId });
+  if (!avg) {
```

Comment:
this will fail in case of no screenshot, the value could be 0

Reply:
okay - fixed it - if there are no screenshots with that launch id it returns null